### PR TITLE
Bump some dependecies to more recent versions

### PR DESCRIPTION
**Pull request by emodric**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | N/A
| **Bug fix**        | no
| **New feature**    | no
| **Target version** | `7.1`
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

Just a bump to more recent versions which properly support Symfony 3.4 and 4.0.

There are no (relevant) breaking changes in newer versions:

- https://github.com/1up-lab/OneupFlysystemBundle/compare/2.0.1...3.0.2
- https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/CHANGELOG.md

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,9 +27,9 @@
         "ocramius/proxy-manager": "^1.0|^2.0",
         "doctrine/doctrine-bundle": "~1.3",
         "liip/imagine-bundle": "~1.0",
-        "oneup/flysystem-bundle": "^1.0|^2.0",
+        "oneup/flysystem-bundle": "^3.0",
         "friendsofsymfony/http-cache-bundle": "~1.2|^1.3.8",
-        "sensio/framework-extra-bundle": "~3.0",
+        "sensio/framework-extra-bundle": "^5.0",
```
Comment (andrerom) [inline composer.json]: Maybe set ^5.1 for the service id name "revert" they did, might affect us if we touch any of the services.

Comment (emodric) [reply composer.json]: Done!

**Comment by emodric** [comment]
@andrerom Also bumped Proxy Manager and Templated URI dependencies.

**Comment by emodric** [comment]
It does :) AppVeyor failure seems unrelated.

**Comment by andrerom** [comment]
It is, timing issue in a test _(feel free to do PR on that, we tend to try to fix them to avoid it happening again, but on some tests it's hard to avoid)_

**Comment by emodric** [comment]
> feel free to do PR on that, we tend to try to fix them to avoid it happening again, but on some tests it's hard to avoid

@MarioBlazek Do you wanna try your luck? :)

**Comment by MarioBlazek** [comment]
@emodric why not 👍

**Pull request merged**